src/components/DeveloperCard.vue

```diff
@@ -25,7 +25,7 @@
 
         <div class="developer-card__description">
             <p class="text-muted text-monospace">
-                <small>{{ developer.summary }}</small>
+                <small class="text-card">{{ developer.summary }}</small>

```

Comment:
Add the `title` attribute to the description, so if the content is hidden we can see it by hovering the description.